tests/driver_bme280/Makefile

```diff
@@ -0,0 +1,15 @@
+APPLICATION = driver_bme280
+include ../Makefile.tests_common
+
+USEMODULE += bme280
+USEMODULE += xtimer
+
+# set default device parameters in case they are undefined
+TEST_I2C                  ?= I2C_0
+TEST_I2C_ADDR             ?= 0x77
+
+# export parameters
+CFLAGS += -DTEST_I2C=$(TEST_I2C)
+CFLAGS += -DTEST_I2C_ADDR=$(TEST_I2C_ADDR)
```

Comment:
no need for the lines above. If you want to use a custom I2C device, you can define it right on the command line setting the define through CFLAGS